terraform/helm/genesis/files/genesis.sh

```diff
@@ -86,7 +86,8 @@ for i in $(seq 0 $(($NUM_VALIDATORS-1))); do
         --username $username \
         --validator-host $validator_host \
         --full-node-host $fullnode_host \
-        --stake-amount $CUR_STAKE_AMOUNT
+        --stake-amount $CUR_STAKE_AMOUNT \
+        --join-during-genesis
```

Comment:
So this actually makes all compat tests fail, since `--join-during-genesis` is not a valid flag in the genesis tooling on `testnet` branch. We'll need to omit `--join-during-genesis`, and make it default to true (when it is expected) @gregnazario @movekevin 

https://github.com/aptos-labs/aptos-core/pull/4569#issuecomment-1259988438 is legitimate tooling compat failure

Reply:
This passes on main now though

Reply:
It only breaks the compatibility test, where the PR branch includes this commit. e.g. it just failed here on my PR: https://github.com/aptos-labs/aptos-core/pull/4590#issuecomment-1260082471

Reply:
fix is on the way... bundling with this PR: https://github.com/aptos-labs/aptos-core/pull/4590